src/core/modules/impl/handler/JsHandlerApi.ts

```diff
@@ -1,4 +1,4 @@
-import { GQLNodeInterface } from 'legacy/gqlResult';
+import { GQLNodeInterface } from '../../../../legacy/gqlResult';
```

Comment:
these weren't causing problems since it's just types, but could create some issues in the future